### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by garcijo**
Implemented middleware to verify that users are authenticated. Also modified some variable names to maintain uniformity in the code.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
+      "tuupola/slim-basic-auth": "^2.0"
     },
     "autoload": {
       "psr-4":{
         "Web\\": "src/Web"
       }
     },
     "require-dev": {
-      "squizlabs/php_codesniffer": "2.*",
       "fabpot/php-cs-fixer": "^1.11"
```
Comment (vzverv) [inline composer.json:31]: now it is recommended to use friendsofphp/php-cs-fixer

**Comment by adamaltman** [comment]
Nice :+1:

File: src/Web/Domain/SpotifyFeed.php [hunk 2]
```diff
@@ -73,6 +73,7 @@ public function getMusic(string $user):array
         $sql = 'SELECT * FROM likes WHERE user=:user';
         $stmt = $this->db->prepare($sql);
         $result = $stmt->execute(['user' => $user]);
+        $songs = array();
```
Comment (dara123) [inline src/Web/Domain/SpotifyFeed.php]: you can use `[]` short array

File: src/Web/Domain/UserMapper.php [hunk 3]
```diff
@@ -28,14 +28,14 @@ public function loginUser(string $user_email, string $user_pass):UserEntity
...
      *
-     * @param string $user_email The current user's username
+     * @param string $userEmail The current user's username
      */
-    public function searchUser(string $user_email):UserEntity
+    public function searchUser(string $userEmail):UserEntity
     {
-        $email = $user_email;
-        $sql = 'SELECT * FROM user WHERE email = :user_email';
+        $email = $userEmail;
```
Comment (dara123) [inline src/Web/Domain/UserMapper.php]: why need to copy this variable?

**Pull request merged**